scripts/benchmark_bolt.py

```
@@ -0,0 +1,56 @@
+from datetime import datetime, timedelta
+
+from api_server import app, get_db
+
+
+def setup_dummy_data(num_crew=50):
+    with app.app_context():
+        conn = get_db()
+        conn.execute("DELETE FROM crew_availability")
+        conn.execute("DELETE FROM crew")
+
+        now = datetime.now()
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)